Bio/SeqIO/__init__.py

```diff
@@ -1048,15 +1048,15 @@ def proxy_factory(format, filename=None):
 }
 
 
-def convert(in_file, in_format, out_file, out_format, alphabet=None):
+def convert(in_file, in_format, out_file, out_format, mol_type=None):
     """Convert between two sequence file formats, return number of records.
 
     Arguments:
      - in_file - an input handle or filename
      - in_format - input file format, lower case string
      - out_file - an output handle or filename
      - out_format - output file format, lower case string
-     - alphabet - optional alphabet to assume
+     - mol_type - optional molecule type to apply: "DNA", "RNA" or "protein".
```

Comment:
perhaps we should call this argument `molecule_type` for consistency

Reply:
OK.